json-to-openapi-converter/json-to-yaml.py

```diff
@@ -121,6 +121,7 @@ def __init__( self, input, output, overwrite, base_file, service_file, odata_sch
                     out_filename_short = filename.rsplit( ".", 1 )[0] + ".yaml"
                     if len( [ i for i in do_not_write if out_filename_short.startswith( i ) ] ) == 0:
                         if overwrite or is_unversioned( filename ) or ( not os.path.isfile( out_filename ) ):
+                            yaml.Dumper.ignore_aliases = lambda *args : True

```

Comment:
This was needed to avoid the insertion of awkward "id00X" anchors when it detects duplications in the output. The confusion by a human looking at the output is not worth the space savings.